src/textual/_config.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import annotations
+
+from pathlib import PurePath
+from typing import Iterable, Collection
+
+import tomli
```

Comment:
I guess this will become something like this when Python 3.11 is out? :slightly_smiling_face: 
```py
try:
  import tomllib
except ImportError:
  import tomli as tomllib
```
(or using a Python version conditional statement ofc)

Reply:
Yup, exactly!